packages/protocol/contracts/bridge/libs/LibBridgeRelease.sol

```diff
@@ -9,19 +9,29 @@ pragma solidity ^0.8.20;
 import { AddressResolver } from "../../common/AddressResolver.sol";
 import { EtherVault } from "../EtherVault.sol";
 import { IBridge } from "../IBridge.sol";
+import {
+    IERC165Upgradeable
+} from "@openzeppelin/contracts-upgradeable/utils/introspection/IERC165Upgradeable.sol";
 import { LibBridgeData } from "./LibBridgeData.sol";
 import { LibBridgeStatus } from "./LibBridgeStatus.sol";
 
+interface VaultContract {
```

Comment:
rename this to `IRecallableMessageSender` and move it to it's own file or to the IBridge.sol.

rename its sole function to "onMessageRecalled" and this function should return the function selector. See IERC1155Receiver in openzapline code.